elasticluster/cluster.py

```diff
@@ -740,6 +744,47 @@ def stop(self, force=False, wait=False):
         else:
             self._delete_saved_data()
 
+    def pause(self):
+        """Pause all VMs in this cluster and store data so that they
+        can be restarted later.
+        """
+        log.debug("Pausing cluster `%s`", self.name)
```

Comment:
I'd raise this to severity `info`:
```suggestion
        log.info("Pausing cluster `%s` ...", self.name)
```